torch/_inductor/mkldnn_lowerings.py

```diff
@@ -40,21 +40,26 @@ def create_int8_compensation(
     weight_compens = None
     x_w_scale = None
     if all(
-        isinstance(item, ir.TensorBox) and item.get_name() in V.graph.constants
+        isinstance(item, ir.TensorBox)
+        and item.get_name() in V.graph.constants
+        and hasattr(item.data, "data")
+        and isinstance(item.data.data, ir.ConstantBuffer)
         for item in [x_scale, x_zp, w_scale]
     ):
         use_int8_fast_compensation_path = True
         x_w_scale_tensor = (
             V.graph.constants[x_scale.get_name()]
             * V.graph.constants[w_scale.get_name()]
         )
+        x_w_scale_tensor.squeeze_()
```

Comment:
This changes seem fine. But since it's un-related to this failure, I think we had better remove it in this PR.

Reply:
Ok. It's removed